libafl/src/feedbacks/map.rs

```diff
@@ -424,12 +424,16 @@ where
     fn is_interesting(
         &mut self,
         state: &mut S,
-        manager: &mut EM,
-        input: &I,
+        _manager: &mut EM,
+        _input: &<S as UsesInput>::Input,
```

Comment:
don't use <S as UsesInput>::Input,
feedbacks is already 99% Use*-free module

Reply:
What else? Do I bring back `I`?

Reply:
Changed it... a bit?

Reply:
I is fine